scripts/cm_test_all_sandia

```diff
@@ -397,6 +442,13 @@ elif [ "$MACHINE" = "white" ]; then
                "cuda/9.2.88 $CUDA_MODULE_LIST $CUDA_IBM_BUILD_LIST ${KOKKOS_PATH}/bin/nvcc_wrapper $CUDA_WARNING_FLAGS"
                "cuda/10.1.105 $CUDA_MODULE_LIST $CUDA_IBM_BUILD_LIST ${KOKKOS_PATH}/bin/nvcc_wrapper $CUDA_WARNING_FLAGS"
     )
+  elif [ "$SPOT_CHECK_TPLS" = "True" ]; then
```

Comment:
OK, I see here that the modules are not the same.

Reply:
Yes, the modules differ. This can all be improved, but is a functional first pass to get nightly spot-check added with TPLs on the machines that don't have a BLAS on the system path.